<ui version="4.0" >
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="res/res.qrc" >
    <normaloff>:/xcalc.svg</normaloff>:/xcalc.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget" >
   <layout class="QGridLayout" name="gridLayout_2" >
    <item row="0" column="0" >
     <widget class="QTabWidget" name="tabWidget" >
      <property name="tabPosition" >
       <enum>QTabWidget::West</enum>
      </property>
      <property name="tabShape" >
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex" >
       <number>2</number>
      </property>
      <widget class="CalcPanel" name="tab_3" >
       <attribute name="title" >
        <string>xCalc</string>
       </attribute>
      </widget>
      <widget class="IvaPanel" name="tab_2" >
       <attribute name="title" >
        <string>IVA</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>Preferences</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout" >
        <property name="sizeConstraint" >
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>0</number>
        </property>
        <item row="0" column="0" >
         <widget class="QToolBox" name="toolBox" >
          <property name="currentIndex" >
           <number>1</number>
          </property>
          <property name="tabSpacing" >
           <number>0</number>
          </property>
          <widget class="QWidget" name="page" >
           <property name="geometry" >
            <rect>
             <x>0</x>
             <y>0</y>
             <width>303</width>
             <height>250</height>
            </rect>
           </property>
           <attribute name="label" >
            <string>Programa</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_5" >
            <item row="0" column="0" >
             <widget class="QCheckBox" name="chAlwaysOnTop" >
              <property name="text" >
               <string>Sempre visivel no topo?</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <spacer name="verticalSpacer_2" >
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>20</width>
                <height>205</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="SerialPanel" name="page_2" >
           <property name="geometry" >
            <rect>
             <x>0</x>
             <y>0</y>
             <width>303</width>
             <height>250</height>
            </rect>
           </property>
           <attribute name="label" >
            <string>Porta serial</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_4" />
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>348</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>Ficheiro</string>
    </property>
    <addaction name="actionAbrirGaveta" />
    <addaction name="actionClose" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <addaction name="menuFile" />
  </widget>
  <widget class="QToolBar" name="mainToolBar" >
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionAbrirGaveta" />
   <addaction name="separator" />
   <addaction name="actionClose" />
   <addaction name="actionQuit" />
  </widget>
  <widget class="QStatusBar" name="statusBar" />
  <action name="actionClose" >
   <property name="icon" >
    <iconset resource="res/res.qrc" >
     <normaloff>:/dialog-close.svg</normaloff>:/dialog-close.svg</iconset>
   </property>
   <property name="text" >
    <string>Fechar</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="icon" >
    <iconset resource="res/res.qrc" >
     <normaloff>:/application-exit.svg</normaloff>:/application-exit.svg</iconset>
   </property>
   <property name="text" >
    <string>Sair</string>
   </property>
  </action>
  <action name="actionRestore" >
   <property name="icon" >
    <iconset resource="res/res.qrc" >
     <normaloff>:/view-fullscreen.svg</normaloff>:/view-fullscreen.svg</iconset>
   </property>
   <property name="text" >
    <string>Restore</string>
   </property>
  </action>
  <action name="actionAbrirGaveta" >
   <property name="icon" >
    <iconset resource="res/res.qrc" >
     <normaloff>:/mail-receive.svg</normaloff>:/mail-receive.svg</iconset>
   </property>
   <property name="text" >
    <string>Abrir Gaveta</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>IvaPanel</class>
   <extends>QWidget</extends>
   <header location="global" >ivapanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CalcPanel</class>
   <extends>QWidget</extends>
   <header location="global" >calcpanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SerialPanel</class>
   <extends>QWidget</extends>
   <header location="global" >serialpanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res/res.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>119</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRestore</sender>
   <signal>triggered()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>showNormal()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>166</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>quit()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>166</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>quit()</slot>
 </slots>
</ui>
